tests/test_logs.py

```diff
@@ -1,380 +1,711 @@
-from unittest import mock
+from datetime import datetime
 
-from rcon.extended_commands import Rcon
+import pytest
 
-#  Unable to parse line: [4:48 min (1659278057)] KICK: [|||™ Marcraƒt] has been kicked. [Host closed the connection.]
-# TODO: Unable to parse line: [4:54 min (1659283019)] KICK: [[BC] Gimrrin] has been kicked. [Host closed the connection.]
```

Comment:
You removed that todo but is this one handled now?

Reply:
That is correct, not sure how to link it directly in here but line 515 on the tests file we check for the `Host closed the connection.` style of disconnects.